server.js

```diff
@@ -2021,6 +2021,51 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Code Climate issues integration
+camp.route(/^\/codeclimate\/issues\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `github/me-and/mdf`.
+  var format = match[2];
+  var options = 'https://codeclimate.com/' + userRepo + 'badges/issue_count.svg'
```

Comment:
You are missing a `/` before `'badges/issue_count.svg'`.
Also, you are missing semicolon at the end of the